dashboard/app/helpers/aitutor_system_prompt_helper.rb

```diff
@@ -0,0 +1,81 @@
+module AitutorSystemPromptHelper
+  def self.get_base_system_prompt
```

Comment:
Ultimately we'll move this back to s3, but during this testing/iterating phase I want it to be in a place that can be easily referenced by levelbuilders & Product so they can copy and paste bits.